Examples/Scripts/Python/seeding.py

```diff
@@ -68,13 +68,70 @@ def __call__(self, parser, namespace, values, option_string=None):
         "minPt",
         "bFieldInZ",
         "impactMax",
+        "interactionPointCut",
+        "arithmeticAverageCotTheta",
+        "deltaZMax",
+        "maxPtScattering",
+        "zBinEdges",
+        "skipPreviousTopSP",
+        "zBinsCustomLooping",
+        "rRangeMiddleSP",
+        "useVariableMiddleSPRange",
+        "binSizeR",
+        "forceRadialSorting",
+        "seedConfirmation",
+        "centralSeedConfirmationRange",
+        "forwardSeedConfirmationRange",
         "deltaR",  # (min,max)
+        "deltaRBottomSP",  # (min,max)
+        "deltaRTopSP",  # (min,max)
+        "deltaRMiddleSPRange",  # (min,max)
         "collisionRegion",  # (min,max)
         "r",  # (min,max)
         "z",  # (min,max)
         "beamPos",  # (x,y)
     ],
-    defaults=[None] * 7 + [(None, None)] * 5,
+    defaults=[None] * 21 + [(None, None)] * 8,
```

Comment:
uhm... what was the meaning of this before? what is the meaning of this now? 😅

Reply:
This is an argument to namedtuple() which is setting the default for each element to None. 
Since some of the elements are truples, for example "collisionRegion" which accepts (min,max), we need to set the default values to [(None, None)].